### ISSUE TITLE
Missing important npm subcommands

### ISSUE THREAD

**Issue by Enet4** [typesafehub/npm#28]
It appears that `Npm` only provides the `update` subcommand, but the currently maintained versions of npm (v6 and v7) support other important commands. Namely:

- `install` is the preferred subcommand for preparing a project to be used, installing any missing modules and running preparation scripts.
- `ci` also works for this purpose, creating a fresh node_modules folder each time and checking that the package-lock is consistent with the package.json.

This has been brought up in sbt/sbt-js-engine#51, which would benefit from calling `npm install` instead of `npm update`.

If there is something that I can do to assist in this matter, I would be glad to know. My main limitation is that I'm fairly inexperienced in Scala.

**Comment by mkurz** [comment]
@Enet4 Do you think you are able to provide a pull request?

**Comment by Enet4** [comment]
Yes, I will try, @mkurz. But from what I could gather, the existing tests do not pass in the current revision (#27). It might be worth fixing this before admitting any new features.

**Comment by mkurz** [comment]
Yeah, unfortunately right now I don't have much time... Maybe in a couple of weeks or month I can work on that...
